### ISSUE TITLE
Version control

### ISSUE THREAD

**Issue by favoyang** [met44/uStableObject#1]
Hi @met44,

Thanks for making this awesome UPM package. I'm the creator of the OpenUPM platform, an open-source UPM registry, and continuous build service based on Git tags. It's hosting more than 200+ open-source UPM packages at the time of writing.

To make it happen, I would suggest you make GitHub releases (which create Git tags) for version control. It can be achieved either manually or using [an automatic approach](https://medium.com/openupm/how-to-maintain-upm-package-part-2-f352fbf5f87c).

After your GitHub release, our build pipelines will detect the changes and build UPM packages in 5-10 minutes. Then you can visit your package at https://openupm.com/packages/com.met44.ustableobject/.

Hopeful this makes sense for you.

**Comment by met44** [comment]
Hey @favoyang, can you summarize what you'd like me to do ? I'm swamped with too many project and cannot go through your doc at this time. 

Is there some sort of file you'd like me to include to make the project compatible with your platform ?

**Comment by favoyang** [comment]
Basically, our build pipelines scan new valid Git tags every 5 minutes to build releases. A valid Git tag can be `vx.y.z`, `x.y.z`, or `upm/x.y.z`.

***I'd like you to create [GitHub release](https://github.com/met44/uStableObject/releases) (which create Git tags), so our build pipelines can catch up.***

Furthermore, considering the extra work to create Git tags, bump version for each release... I also suggest using GitHub Actions to [automatic the process](https://medium.com/openupm/how-to-maintain-upm-package-part-2-f352fbf5f87c). It's worth to read, but requires to change your commit message style, and take time to setup. If you're busy, you can just create a Git tag manually for now.
